### PR TITLE
feat: Add note about MinK system accounts

### PR DESCRIPTION
**Pull request by cloudjumpercat**
### Description

MinK auto creates a system account for zones. This shows up in the UI but is read only, so we needed to add a note about it so customers know what it does and why it's there. 
 

DOCU-3467
### Testing instructions

Preview link: 

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

File: app/konnect/mesh-manager/service-mesh.md
Comment (johnharris85) [inline app/konnect/mesh-manager/service-mesh.md:27]:
```suggestion
    > **Note:** Mesh Manager automatically creates a [managed service account](/konnect/org-management/system-accounts/) that is only used in the zone creation process. This managed service account cannot be edited or deleted manually. Instead, its life-cycle is managed by {{site.konnect_short_name}} and it is deleted automatically when the zone is deleted.
```

File: app/konnect/org-management/system-accounts.md
Comment (johnharris85) [inline app/konnect/org-management/system-accounts.md:19]:
```suggestion
[Mesh Manager](/konnect/mesh-manager/) automatically creates a managed service account that is only used in the zone creation process. This managed service account cannot be edited or deleted manually. Instead, its life-cycle is managed by {{site.konnect_short_name}} and it is deleted automatically when the zone is deleted.
```

**Review by johnharris85 — CHANGES_REQUESTED** [review]
Made a couple of wording changes (it's not really _special_, just a new type of service account). I _think_ I'd be in favor of adding a sub-heading on the service account page called `Managed Service Accounts` to make folks clear that they're a first-class _thing_, maybe calling out that there's a flag in the API that denotes this type. But as Mesh is the only thing that uses them right now, I'm also OK if they just stay as a note on that page I guess? @haydenlam probably has final say / approval on that one.

**Comment by johnharris85** [comment]
> @johnharris85 One question, what is the exact API flag for managed system accounts? I currently just have `blank` there because I wasn't sure what it was.

It looks like it's `konnect_managed` based on this: https://docs.konghq.com/konnect/api/identity-management/latest/#/System%20Accounts/get-system-accounts-id

File: app/konnect/org-management/system-accounts.md
Comment (cloudjumpercat) [inline app/konnect/org-management/system-accounts.md:21]:
```suggestion
Managed system accounts are system accounts whose life cycle is managed by {{site.konnect_short_name}} instead of the user. The `konnect_managed: true` flag in the API denotes this type of system account.
```

**Review by lena-larionova — APPROVED** [review]
lgtm!

**Pull request merged**
